File: Iniciante/1002 - Área do Círculo/C#/1002 - Área do Círculo.cs
using System;
using System.Globalization; // formata os valores para en-us (.)

class URI {
    static void Main() {
        double raio, area;
        double pi = 3.14159;

        raio = double.Parse(Console.ReadLine(), CultureInfo.InvariantCulture);
        area = Math.Pow(raio, 2)*pi; // Math.Pow(base, expoente)

        Console.WriteLine("A=" + area.ToString("0.0000", System.Globalization.CultureInfo.InvariantCulture));

    }
}

File: Iniciante/1002 - Área do Círculo/Java/1002 - Área do Círculo.java
import java.util.Locale;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner leitor = new Scanner(System.in).useLocale(Locale.US);
		
		double raio = leitor.nextDouble();
		double area = Math.pow(raio, 2) * 3.14159;
		
		System.out.printf("A=%.4f", area);

	}

}